avm/utilities/pipelines/sharedScripts/helper/Get-CrossReferencedModuleList.ps1

```diff
@@ -93,6 +93,7 @@ function Get-ReferenceObject {
 
         $resultSet.resourceReferences += @() + $moduleContent | Where-Object { $_ -match "^resource .+ '(.+?)' .+$" } | ForEach-Object { $matches[1] }
         $resultSet.remoteReferences += @() + $moduleContent | Where-Object { $_ -match "^module .+ '(.+:.+?)' .+$" } | ForEach-Object { $matches[1] }
+        $resultSet.remoteReferences += @() + $moduleContent | Where-Object { $_ -match "^import .+ '(.+:.+?)'$" } | ForEach-Object { $matches[1] }
```

Comment:
kept in a different line in case we'd like to change the cross-referenced "Type" to something different, e.g. Imported remote reference or something If not I'd probably merge with the above line

Reply:
TBD: should we add local import references as well? i.e. `import {.*} from 'modules\...`'